index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+
+<head>
+    <html lang="en">
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <script defer src="script.js"></script>
+    <script defer src="function.js"></script>
+    <title>Weather App</title>
+</head>
+
+<body>
+
+    <header>
+    <!--Header for Weather App-->
+        <h1>Weather App</h1>
+        <br>
+        <form class="search">
+            <!--City Search Form-->
+            <label for="location">Pick a Location</label>
+            <input type="text" name="location" input id="location" required />
+            <br>
+            <input type="submit" name="location" value="Get Weather" />
+        </form>
+    </header>
+    <br>
+
+    <div class="weather">
+    <aside class="conversion" hidden>
+        <form class="temperature-conversion">
+            <!--Temperature Conversion Widget-->
+           <p>Convert the Temperature:</p>
+          
+            <input type="text" list="numbers" />
+            <datalist id="numbers">
+               <option></option>
+            </datalist>
+            <br>
+            <label for="Celsius">To Celsius</label><input type="radio" name="convert" value="To Celsius"
+                id="Celsius"><br>
+            <label for="Fahrenheit">To Fahrenheit</label>
+            <input type="radio" name="convert" value="To Fahrenheit" id="Fahrenheit">
+        </form>
+      
+    </aside>
+    <main class="view-weather">
+        <!--Contains current weather-->
+        <section class="current">
+        <p class="empty">Choose a location to view the weather</p>
+        <article class="current" hidden />Current Forecast<br>
```

Comment:
i believe this hidden element is contributing to you failing some of the tests. I'm not sure what the original intent was here but if it's not necessary it might be worth removing.